### DISCUSSION TITLE
How to turn off scroll bar dark mode?

### DISCUSSION THREAD

**Discussion by DeFaLT-dj** [darkreader/darkreader#8035] [Q&A] [answered]
Microsoft Edge added an overlay scroll bar flag a few months ago and it worked fine. But I realized some websites still use old style scroll bars in some websites and I thought the feature isn't done yet so not all websites adapting it. Then I turned off the dark reader and realized it is actually dark reader overrides it. I can understand this behavior because old scroll bar used use white color regardless of the web pages so this can be so useful for other browsers. But I want to turn this setting off. Is there a way to do it?

This is the scroll bar with darkreader

![image](https://user-images.githubusercontent.com/64617514/150686902-247eeeb0-519d-40da-8db3-e406de3862ca.png)

This is the scroll bar when dark reader is turned off (this is how it supposed to look like)

![image](https://user-images.githubusercontent.com/64617514/150686916-3356dda6-62c6-4e27-8b54-afb3c50b0959.png)

**Comment by Gusted** [comment, marked as answer]
See https://github.com/darkreader/darkreader/issues/858#issuecomment-887061151
